### DISCUSSION TITLE
How to disable eslint for jsx in vue3

### DISCUSSION THREAD

**Discussion by Jokcy** [codesandbox/codesandbox-client#5260] [Q&A]
https://codesandbox.io/s/misty-sun-1r0pd?file=/src/main.js

See this sandbox, build success but the editor always show `Parsing error: Unexpected token < (null)eslint`

How can I disable it?

**Comment by SaraVieira** [comment]
Hey!

Sorry for the error, seems somewhere that between vetur and eslint it's not letting me disable the line.

We already have the ability to add custom eslint settings in our backlog but no date :/

**Reply by Jokcy** [reply]
so do I need to create an issue for this?

**Reply by SaraVieira** [reply]
No need, there are some and it's already in our backlog
